app/main.py

```diff
@@ -57,9 +79,71 @@ async def favicon():
 def get_language_code(language: Language | str):
     return type(language) is str and language[:2] or language.value
 
+if PROXY_PANEL_THROUGH_FASTAPI:
+    # Forward the websockets through a bridge
+    from fastapi import WebSocket
```

Comment:
I don't use conditional imports very often, so I wonder what the best place for them are, together with the rest of the imports at the beginning of the file or here?

Reply:
we spare resource when they are not required to be loaded... But I may group them after the if directly